numba_rvsdg/core/datastructures/scfg.py

```diff
@@ -270,26 +494,92 @@ def _insert_block(
     def insert_SyntheticExit(
         self, new_name: str, predecessors: Set[str], successors: Set[str],
     ):
+        """Inserts a synthetic exit block into the control flow graph.
+
+        Parameters
+        ----------
+        new_name: str
+            The name of the newly created block.
+        predecessors: Set[str]
+            The set of names of BasicBlock that act as predecessors
+            for the block to be inserted.
+        successors: Set[str]
+            The set of names of BasicBlock that act as successors
+            for the block to be inserted.
+        """
         self._insert_block(new_name, predecessors, successors, SyntheticExit)
 
     def insert_SyntheticTail(
         self, new_name: str, predecessors: Set[str], successors: Set[str],
     ):
+        """Inserts a synthetic tail block into the control flow graph.
+
+        Parameters
+        ----------
+        new_name: str
+            The name of the newly created block.
+        predecessors: Set[str]
+            The set of names of BasicBlock that act as predecessors
+            for the block to be inserted.
+        successors: Set[str]
+            The set of names of BasicBlock that act as successors
+            for the block to be inserted.
+        """
         self._insert_block(new_name, predecessors, successors, SyntheticTail)
 
     def insert_SyntheticReturn(
         self, new_name: str, predecessors: Set[str], successors: Set[str],
     ):
+        """Inserts a synthetic return block into the control flow graph.
+
+        Parameters
```

Comment:
As with `SyntheticExit`.

Reply:
Fixed in [3e369c9](https://github.com/numba/numba-rvsdg/pull/59/commits/3e369c985b3af41872bcee5df9345852d780053f)